grid/js/schematic.js

```diff
@@ -65,12 +67,10 @@ function visualize(error, contribs, contribs2, contribs3, disclosure1, disclosur
     // Initialize the navigation state.
     var defaultSection = "contribution-limits";
     var section = getQueryVariables().section || defaultSection;
-    signal.call("navigate", null, section);
-    d3.selectAll(".nav li")
-        .classed("active", function() {
-            // Set the initial active tab
-            return d3.select(this).select("a").attr("aria-controls") === section;
-          })
+
+    d3.select("a[href='#" + section + "']")
+      .node()
+      .click()
```

Comment:
@curran, precisely -- it sets the active class on the tab itself as well as the appropriate tab-pane (since the tab is just an anchor `<a>` to the tab-pane)